modules/charon-core/src/main/java/org/wso2/charon3/core/schema/SCIMSchemaDefinitions.java

```diff
@@ -1495,4 +1507,18 @@
                     SCIMResourceTypeSchemaDefinition.SCHEMA,
                     SCIMResourceTypeSchemaDefinition.SCHEMA_EXTENSIONS);
 
+    /*
```

Comment:
Follow the commenting pattern.
/\* 
 \* Comment.
 \* /

Reply:
fixed with https://github.com/wso2/charon/pull/323/commits/6a773e3384a5db3baab94370f9f64a904834a48d